docs/user-guide/how_to_use_vgpu.md

```diff
@@ -0,0 +1,195 @@
+# GPU Sharing User guide
+
+## Contact And Bug Report
+
+Thanks staff of 4paradigm.com for contributing this feature to volcano, if you encountered any issues, feel free to submit an issue, or sending an email to <limengxuan@4paradigm.com>
+
+## Main features
+
+***GPU sharing***: Each task can allocate a portion of GPU instead of a whole GPU card, thus GPU can be shared among multiple tasks.
+
+***Device Memory Control***: GPUs can be allocated with certain device memory size (i.e 3000M) or device memory percentage of whole GPU(i.e 50%) and have made it that it does not exceed the boundary.
+
+***Easy to use***: See the examples below.
+
+## Environment setup
+
+### Install volcano
+
+#### 1. Install from source
+
+Refer to [Install Guide](../../installer/README.md) to install volcano.
+
+After installed, update the scheduler configuration:
+
+```shell script
+kubectl edit cm -n volcano-system volcano-scheduler-configmap
+```
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: volcano-scheduler-configmap
+  namespace: volcano-system
+data:
+  volcano-scheduler.conf: |
+    actions: "enqueue, allocate, backfill"
+    tiers:
+    - plugins:
+      - name: priority
+      - name: gang
+      - name: conformance
+    - plugins:
+      - name: drf
+      - name: predicates
+        arguments:
+          predicate.VGPU4pdEnable: true # enable vgpu
```

Comment:
1. If the `vgpu` and `gpu share` cannot be enabled at the same time, a detection should be done when the scheduler starts.
2. I'm not sure if the name of a commercial company like `4pd` can be used directly in cncf projects. @Thor-wl @william-wang Do you have any suggestion

Reply:
1. fixed

Reply:
> 1. If the `vgpu` and `gpu share` cannot be enabled at the same time, a detection should be done when the scheduler starts.
> 2. I'm not sure if the name of a commercial company like `4pd` can be used directly in cncf projects. @Thor-wl @william-wang Do you have any suggestion

Yes, you are right. suggest use a common word like `predicate.VGPUEnable`